net/teleport/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        gravitational teleport 2.2.3 v
+
+homepage            http://gravitational.com/teleport/
+categories          net
+maintainers         { gmail.com:graemelawes @gclawes } openmaintainer
+description         Teleport SSH server and Certificate Authority
+long_description    Teleport is a modern SSH server and CA for managing clusters \
+                    that provides secure login with certificate-based access, \
+                    2-factor authentication, session logging and replay, \
+                    integration with Google Apps and other OAuth identity providers, \
+                    and a Web UI.  Built on the Golang SSH library, and compatible with OpenSSH \
```

Comment:
You need to remove the trailing backslash, else the licence in the next line is treated as comment.

Please run `port lint` next time when submitting files. It would tell you
```
--->  Verifying Portfile for teleport
Warning: no license set
--->  0 errors and 1 warnings found.
```